src/providers/proxy/proxy_init.c

```diff
@@ -76,6 +76,28 @@ static errno_t proxy_id_conf(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static bool local_auth_enabled(TALLOC_CTX *mem_ctx, struct be_ctx *be_ctx)
+{
+    int ret;
+    char *local_policy = NULL;
+    bool res;
+
+    ret = confdb_get_string(be_ctx->cdb, mem_ctx, be_ctx->conf_path,
```

Comment:
NULL is sufficient, you don't need to use mem_ctx since you're not returning any data.

Reply:
changed